### DISCUSSION TITLE
在一个载入就运行的alova实例中，如何处理数据？

### DISCUSSION THREAD

**Discussion by yydsapi** [alovajs/alova#279] [Q&A]
### 请详细描述bug

  setup(props) {
    const {proxy} = getCurrentInstance()
    const {loading, data, error} = useRequest(  // 使用alova实例创建method并传给useRequest即可发送请求
      proxy.conn.Post('/api/oa/org/list', {
        org: "1" //get:params: {id: 1 }
      })
    )

<template>
  <div>
<div v-if="loading">Loading...</div>
    <div v-else-if="error">{{ error.message }}</div>
    <span v-else>{{ JSON.parse(data["org"]) }}
    <el-tree
    :data="data"
    show-checkbox
    node-key="id"
    default-expand-all
    :default-checked-keys="[5]"
    :expand-on-click-node="false"
    :props="defaultProps"
  >

 return {
      data,
      loading,
      error
}

我想使用JSON.parse(data["org"]) ，而不是data ，有什么好办法吗？谢谢！

### 复现步骤

无

### alova 版本

2.16.2

**Comment by JOU-amjs** [comment]
问题类帖子请在discussion提问
你需要将数据字符串化，建议用一个计算属性
```js
const orgStr = computed(() => JSON.parse(data.value.org))
return {
  orgStr,
  ...
}
```
